src/main/java/org/jboss/forge/arquillian/command/AddContainerDependencyStep.java

```diff
@@ -96,18 +99,33 @@ public Result execute(UIExecutionContext context) throws Exception {
       Map<Object, Object> ctx = context.getUIContext().getAttributeMap();
       Container container = (Container)ctx.get(ContainerSetupWizard.CTX_CONTAINER);
       String version = (String)ctx.get(ContainerSetupWizard.CTX_CONTAINER_VERSION);
+      Project project = getSelectedProject(context);
+      ArquillianFacet arquillian = project.getFacet(ArquillianFacet.class);
+      ArquillianConfig config = arquillian.getConfig();
 
-      containerInstaller.installContainer(
-            getSelectedProject(context),
-            container,
-            version,
-            getVersionedDependenciesMap());
+      final String profileId = container.getProfileId();
+
+      if (container. isSupportedByChameleon(version)) {
+         dependencyManager.addChameleonDependency(project);
+         if (config.isContainerWithDefault("true")) {
+            config.addContainer(profileId);
+         } else {
+            config.addContainerWithAttribute(profileId, "default", "true");
+         }
```

Comment:
nopes, we are not using single entry here. We have different entries for each profile in configuration as user should be able to add different configuration options for each containers.

The idea here is any qualifier should have default true property here. We can make it more intelligent later depending on user's choice by asking user to select one profile which he want to make it default